src/NotificationEventMailing.php

```diff
@@ -182,21 +182,24 @@ public static function send(array $data)
 
                 $documents_ids = [];
                 $documents_to_attach = [];
-                if ($is_html || $CFG_GLPI['attach_ticket_documents_to_mail']) {
+                $documents_data = json_decode($current->fields['documents_data'], true);
+                $itemtype = $documents_data['itemtype'] ?? '';
+                $items_id = $documents_data['items_id'] ?? 0;
```

Comment:
This will be a problem for queued notifications that were generated by previous GLPI version. It should probably fallback to the previously computed value.
```suggestion
                $documents_data = json_decode($current->fields['documents_data'], true);
                $itemtype = $documents_data['itemtype'] ?? $current->fields['itemtype'];
                $items_id = $documents_data['items_id'] ?? $current->fields['items_id'];
```